### PR TITLE
Fix 8535

### PR DESCRIPTION
**Pull request by LemonBoy**

### CODE REVIEW

**Comment by Araq** [comment]
I don't feel confident about your removal of the ``postInitProc`` mechanism.

**Comment by LemonBoy** [comment]
> I don't feel confident about your removal of the postInitProc mechanism.

We can always fish it from the depths of the git history if needed. And I think that if it were used it'd suffer from the same scope-related problems as `preInitProc` and would need some adjustments too.

File: compiler/cgen.nim [hunk 1]
```diff
@@ -1210,11 +1224,6 @@ proc newPreInitProc(m: BModule): BProc =
   # little hack so that unique temporaries are generated:
   result.labels = 100_000
 
-proc newPostInitProc(m: BModule): BProc =
-  result = newProc(nil, m)
-  # little hack so that unique temporaries are generated:
```
Comment (Araq) [inline compiler/cgen.nim]: Well I did write this 'little hack' because it's actually used code. I don't know why the tests don't cover it.

Comment (LemonBoy) [reply compiler/cgen.nim]: I think that's just c&p from `newPreInitProc`, no code at all is emitted for this section.

**Pull request merged**